views/template.py

```diff
@@ -1,11 +1,17 @@
 # -*- coding: utf-8 -*-
 room_created_message = "A %s called %s has been successfully created!"
```

Comment:
the C style format specifiers (e.g %s) are deprecated in python 3, try to use the format() method instead